### DISCUSSION TITLE
AMBE3000

### DISCUSSION THREAD

**Discussion by TS6000** [pschatzmann/arduino-audio-tools#1779] [Q&A] [answered]
Hello, I have an AMBE3000 connected to an ESP32

The AMBE3000 writes me "8kHz sample rate 16 bit PCM" into a buffer with 320 byte how can I output this.

**Comment by pschatzmann** [comment, marked as answer]
You just configure the output stream and write the PCM data to it

I suggest you read the Introduction Wiki
